src/Native/System.Native/pal_process.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "../config.h"
+#include "pal_process.h"
+#include <sys/resource.h>
+#include <assert.h>
+
+// Each platform type has it's own RLIMIT values but the same name, so we need
+// to convert our standard types into the platform specific ones.
+int32_t ConvertRlimitResourcesToPlatformValue(RlimitResources value)
```

Comment:
This should be static since we don't export it.

Also, I suggest using the same naming convention as in the errno shim ConvertRlimitResourcesPalToPlatform.